public/lessons/prefix-sums/example-walkthrough.mdx

```diff
@@ -0,0 +1,30 @@
+## Example Walkthrough
+
+Let's do an example using prefix sums for subarray sum query. 
+
+Given is an array `arr` with length $n = 5$, what is subarray sum between indices $a = 1$ to $b = 3$, inclusive:
+<center>
+<table width = "550px">
+<tr><th>Index $i$</th><td>$0$</td><td style={{backgroundColor: "yellow"}}>$1$</td><td style={{backgroundColor: "yellow"}}>$2$</td><td style={{backgroundColor: "yellow"}}>$3$</td><td>$4$</td></tr>
+    <tr><th>$arr[i]$</th><td>$5$</td><td style={{backgroundColor: "lightgreen"}}>$10$</td><td style={{backgroundColor: "lightgreen"}}>$3$</td><td style={{backgroundColor: "lightgreen"}}>$8$</td><td>$14$</td></tr>
+</table>
+</center><br />
```

Comment:
nit: some formatting if possible. here and anywhere else where applicable

Reply:
i got so used to prettier for formatting and did not realize that lessons are not formatted using prettier. will fix that